docstore/doc.go

```diff
@@ -236,26 +239,26 @@
 // method until it returns io.EOF. Always call Stop when you are finished with an
 // iterator. It is wise to use a defer statement for this.
 //
-//     iter := coll.Query().Where("size", ">", 10).Limit(5).Get(ctx)
-//     defer iter.Stop()
-//     for {
-//         m := map[string]interface{}{}
-//         err := iter.Next(ctx, m)
-//         if err == io.EOF {
-//             break
-//         }
-//         if err != nil {
-//             return err
-//         }
-//         fmt.Println(m)
-//     }
+//   iter := coll.Query().Where("size", ">", 10).Limit(5).Get(ctx)
+//   defer iter.Stop()
+//   for {
+//       m := map[string]interface{}{}
+//       err := iter.Next(ctx, m)
+//       if err == io.EOF {
+//           break
+//       }
+//       if err != nil {
+//           return err
+//       }
+//       fmt.Println(m)
+//   }
 //
 //
 // Errors
 //
 // The errors returned from this package can be inspected in several ways:
 //
-// The Code function from gocloud.dev/gcerrors will return an error code, also
+// The Code function from https://gocloud.dev/gcerrors will return an error code, also
```

Comment:
Cool, that link redirects to godoc. TIL!

Reply:
I was excited to learn this too :)